presto-main/src/test/java/io/prestosql/sql/planner/TestLogicalPlanner.java

```diff
@@ -122,6 +124,13 @@
 public class TestLogicalPlanner
         extends BasePlanTest
 {
+    public TestLogicalPlanner()
+    {
+        super(ImmutableMap.of(
+                JOIN_REORDERING_STRATEGY, JoinReorderingStrategy.NONE.name(),
```

Comment:
This class is a general planner test. It should use default settings.
Some test methods, however, would fail.

Let's introduce `noJoinReordering` method and use it in these tests.

Reply:
Got it.